### DISCUSSION TITLE
28/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#193] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer estuvimos terminando detalles del trabajo de tecnología. 
Hoy empezaré a revisar lo que nos queda del cierre del sprint con android y el trabajo de emprendedurismo. 
El tiempo se pasa volando..

**Comment by rpgrca** [comment]
Ayer terminamos con la entrega de T&D. Hoy tengo clases de emprendedurismo, voy a empezar a leer algo para el script final y el cuestionario y por ahí mañana a la mañana continúo con la parte de Android.

**Comment by fmdcba** [comment]
Ayer con la entrega de tecno y hoy tuve un dia bastante ocupado. No creo que llegue presenciar la clase aunque me hubiera gustado. Luego voy a estar viendo el punto de lean startup para ir cerrando con el trabajo de emprendedurismo.
